CONTRIBUTING.md

```diff
@@ -71,6 +71,53 @@ When afraid to touch force and risk losing your work (do backups!..),
 try *merging master into your branch* instead of rebasing onto it.
 This is discouraged, as it produces ugly hard-to-maintain commit history.
 
+## Commit message rules
+
+When commiting your changes into the repository you may want to follow some
+rules to make the git history more readable and consistent:
+
+* Start the message capitalized (only the first character must be in uppercase)
+* Be short and concise, the whole concept must fit one line
+* If a command is inlined, use backticks
+* Add a double-hashtag if the change matters for the changelog (See below)
+* If the commit fixes a bug start with 'Fix #number - '
+* For extra details, add an empty line and use asterisk item list below
+* Use present simple grammar tense (Add vs Added, Fix vs Fixed/Fixes)
+* Prefix with `WIP:` if it's not yet ready to be merged
```

Comment:
Isn't this just something about PRs and not about `Commit message rules`?

Reply:
Yes 13b8cc4a0067cc17b08d11ae403bcc44d0a8ee50